doc/en/user/source/extensions/css/workshop/point.rst

```diff
@@ -0,0 +1,708 @@
+
+#. Additional properties are available to control a mark's presentation:
+
+   The **mark-size** property is used to control symbol size.
+
+   The **mark-rotation** property controls orientation, accepting input in degrees.
+   
+   Trying these two settings together:
+
+   .. code-block:: css
+
+      [ SCALERANK < 1 ] {
+        mark: symbol(square);
+        mark-size: 8;
+        mark-rotation: 45;
+      }
+
+#. Results in each location being marked with a diamond:
+   
+   .. image:: img/point_mark_3.png
+
+#. Now that we have assigned our point location a symbol we can make use of a **pseudo-selector** to style the resulting shape.
+
+   **:symbol** - provides styling for all the symbols in the CSS document.
+
+   **:mark** - provides styling for all the mark symbols in the CSS document.
+   
+   This form of pseudo-selector is used for all marks:
+
+   .. code-block:: css
+
+      [ SCALERANK < 1 ] {
+        mark: symbol(square);
+        mark-size: 8;
+        mark-rotation: 45;
+      }
+      :mark{
+         fill: white;
+         stroke: black;
+      }
+
+#. Updating the mark to a white square with a black outline.
+
+   .. image:: img/point_mark_4.png
+
+#. The second approach is used to individual configure symbols in the same document.
+
+   **:nth-symbol(1)** - if needed we could specify which symbol in the document we wish to modify.
+
+   **:nth-mark(1)** - provides styling for the first mark symbol in the CSS document.
+   
+   Using this approach marks can be composed of multiple symbols, each with its own settings:
+
+   .. code-block:: css
+
+      [ SCALERANK < 1 ] {
+        mark: symbol(square),symbol(cross);
+        mark-size: 16,14;
+        mark-rotation: 0,45;
+      }
+      :nth-mark(1){
+         fill: red;
+         stroke: black;
+      }
+      :nth-mark(2){
+         fill: black;
+         stroke: white;
+      }
+
+#. Producing an interesting compound symbol effect:
+
+   .. image:: img/point_mark_5.png
+
+Graphic
+-------
+
+Symbols can also be supplied by an external graphic,
+
+.. image:: img/Point_Graphic.svg
+
+This technique was shown with the initial file:`airport.svg` CSS example.
+
+#. To use an external graphic two pieces of information are required.
+
+   **mark** property is defined with a **url** reference to image.
+   
+   **mark-mime** property is used to tell the rendering engine what file format to expect
+   
+   This technique is used to reference files placed in the styles directory.
+    
+   .. code-block:: css
+
+      [ SCALERANK < 1 ] {
+        mark: url(port.svg);
+        mark-mime: "image/svg";
+      }
+
+#. Drawing the provided shape in each location:
+
+   .. image:: img/point_graphic_1.png
+
+#. The **mark** property **url** reference can also be used to reference external images. We can make use of the GeoServer logo.
+
+   .. code-block:: css
+
+      [ SCALERANK < 1 ] {
+           mark: url("http://localhost:8080/img/geoserver_suite_32.png");
```

Comment:
Hmmmmm, yeah good point. It might be there, but might not, I have no idea. Plus it might change with version.

This link may also die or change, but I linked it to the docs' GeoServer logo instead.